app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/helm/BakeHelmConfigForm.tsx

```diff
@@ -21,6 +22,13 @@ export class BakeHelmConfigForm extends React.Component<IFormikStageConfigInject
     ArtifactTypePatterns.HTTP_FILE,
   );
 
+  constructor(props: {}) {
```

Comment:
Would drop the constructor entirely. You can just declare the component's initial state like this:

```
class BakeHelmConfigForm {
  state = { gitRepoArtifactAccounts: [] };
}
```

Reply:
Can't comment on an unchanged line 👎 👎 👎 

You'll need to add a type that looks like this:

```
interface IBakeHelmConfigFormState {
  githubRepoArtifactAccounts: IAccount[];
}
```

and then add it as a type parameter to the component:

```
class BakeHelmConfigForm extends React.Component<IFormikStageConfigInjectedProps, IBakeHelmConfigFormState> {
...
}
```

Reply:
https://github.com/spinnaker/deck/pull/8774/commits/ef5556034375a7259eba9a6ab2997b6262717873 attempts to do this.